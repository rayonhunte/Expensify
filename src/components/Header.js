import React from 'react';
import {NavLink} from 'react-router-dom'
const Header = () => (
  <div>
    <header>
    <h1>Expensify</h1>
    <NavLink to="/" activeClassName="is-active" exact={true}>Home</NavLink>
    <NavLink to="create" activeClassName="is-active">Create</NavLink>
    <NavLink to="/help" activeClassName="is-active">Help</NavLink>
    </header>
  </div>
);

export default Header;