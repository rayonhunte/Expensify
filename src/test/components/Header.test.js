//react-test-render
import React from 'react';
import { shallow } from 'enzyme'
import Header from '../../components/Header';
import toJSON from 'enzyme-to-json'


test('should render header', ()=>{
  const wrapper = shallow(<Header/>)
  expect(wrapper).toMatchSnapshot();
} )