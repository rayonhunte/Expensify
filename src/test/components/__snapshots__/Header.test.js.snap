// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header 1`] = `
<div>
  <header>
    <h1>
      Expensify
    </h1>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      exact={true}
      to="/"
    >
      Dashboard
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="create"
    >
      Create
    </NavLink>
    <NavLink
      activeClassName="is-active"
      ariaCurrent="true"
      to="/help"
    >
      ExpenseHelp
    </NavLink>
  </header>
</div>
`;
